site/content/en/docs/manual/advanced/organization.md

```diff
@@ -5,57 +5,128 @@ weight: 2
-  sees all the tasks and the projects created by other members of the organization,
-  has full access to the `Cloud Storages` feature, and can modify members and their roles.
-- `Owner` - a role assigned to the creator of the organization with maximum capabilities.
+3. Click **Ok**.
 
-In addition to roles, there are groups of users that are configured on the `Admin page`.
-Read more about the roles in [IAM system roles section](/docs/administration/advanced/iam_system_roles).
+The **Invite Members** form has the following fields:
 
-![](/images/image236.jpg)
+![](/images/invitemembers.jpg)
 
-After you add members, they will appear on your organization settings page,
-with each member listed and information about invitation details.
-You can change a member's role or remove a member at any time.
+<!--lint disable maximum-line-length-->
+
+| Field               | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| ------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Email**           | Specifies the email address of the user who is being added to the **Organization**.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
Let's add a piece of information, that this user already must have account on the corresponding instance.
It does not work for any entered email